### PR TITLE
⚡ Implemented automatic position detection for credential fields.

### PR DESCRIPTION
**Pull request by MiloradFilipovic**
Credentials field position is now automatically determined based on authentication fields position:
1. Credentials show up after auth field
2. If there are multiple credentials, they are shown after the last authentication field

### CODE REVIEW

File: packages/editor-ui/src/components/ParameterInputList.vue [hunk 1]
```diff
@@ -130,10 +131,22 @@ export default mixins(
 			node (): INodeUi {
 				return this.$store.getters.activeNode;
 			},
+			nodeType () : INodeTypeDescription {
```
Comment (mutdmour) [inline packages/editor-ui/src/components/ParameterInputList.vue:134]: this can be deleted as well

**Review by mutdmour — APPROVED** [review]

File: packages/editor-ui/src/components/ParameterInputList.vue [hunk 2]
```diff
@@ -131,12 +130,30 @@ export default mixins(
...
+				});
+
+				return index < this.filteredParameters.length ?
+					index : this.filteredParameters.length - 1;
 			},
 		},
 		methods: {
+			getCredentialsDependencies() {
+				const dependencies = new Set();
+				const nodeType = this.$store.getters.nodeType(this.node.type, this.node.typeVersion);
```
Comment (mutdmour) [inline packages/editor-ui/src/components/ParameterInputList.vue:149]: need to address unknown types here
```suggestion
				const nodeType = this.$store.getters.nodeType(this.node.type, this.node.typeVersion) as INodeType | undefined;
```
Bildschirmfoto 2022-06-07 um 14 09 04

Comment (mutdmour) [reply packages/editor-ui/src/components/ParameterInputList.vue:149]: also breaking nodes without creds

**Review by mutdmour — APPROVED** [review]
thanks for addressing my endless feedback :)

**Comment by janober** [comment]
Great work. Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.180.0`

**Pull request merged**